src/ActionModal/components/UndelegationModal.vue

```diff
@@ -294,10 +301,15 @@ export default {
   validations() {
     return {
       amount: {
-        required: (x) => !!x && x !== `0`,
+        required: (x) =>
+          this.currentNetwork.network_type === "polkadot" || (!!x && x !== `0`),
         decimal,
-        max: (x) => Number(x) <= this.maximum,
-        min: (x) => Number(x) >= SMALLEST,
+        max: (x) =>
+          this.currentNetwork.network_type === "polkadot" ||
```

Comment:
max does apply for any network, doesn't it?

Reply:
Yes, I also don't understand this distinction actually